### ISSUE TITLE
`ComposeRedundantComposable` false positive on composable lambda invocation

### ISSUE THREAD

**Issue by JakeWharton** [slackhq/compose-lints#614]
```kotlin
val Lambda = @Composable { }

@Composable
fun Test() {
    Lambda()
}
```
with `com.slack.lint.compose:compose-lint-checks:1.5.4`
